tests/unit/test-mode-comprehensive-validation.test.js

```
@@ -0,0 +1,799 @@
+  const executionTime = Date.now() - startTime;
+  return { result, executionTime };
+}
+
+// Export test configuration and utilities for other test files
+export {
+  TEST_CONFIG,
+  createAdminToken,
+  cleanupAllTestData,
+  measureMemoryUsage,
+  measureExecutionTime
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)